src/hip/hip_build_utils.cpp

```diff
@@ -193,7 +193,7 @@ boost::filesystem::path HipBuild(boost::optional<TmpDir>& tmp_dir,
 
         // call clang-offload-bundler
         tmp_dir->Execute(MIOPEN_OFFLOADBUNDLER_BIN,
-                         "--type=o --targets=hip-amdgcn-amd-amdhsa-" + dev_name + " --inputs=" +
+                         "--type=o --targets=hip-amdgcn-amd-amdhsa--" + dev_name + " --inputs=" +
```

Comment:
Is this change backwards compatible?

Reply:
Yeah IDK, good question.

Reply:
backward compatibility means old executable can run on new runtime. we do not guarantee new executable can run on old runtime. Does MIOpen develop branch promise to work on all ROCm release?

one strange thing is that how could this issue happen? the compiler change passes ePSDB which has MIOpen test. Does this issue only happen to develop branch of this repo?

Reply:
This is already implemented in #674, which provides backward compatibility.